docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,500 @@
+    - `OSN_TLS_SIGN_CERT` is the orderer signed certificate from the TLS CA.
+    - `OSN_TLS_PRIVATE_KEY` is the orderer private key from the TLS CA.
+
+    **Note:** The connection between the `osnadmin` CLI and the orderer requires mutual TLS, although it is not required for the network itself. This means you need to pass the `--client-cert` and `--client-key` parameters on each `osdadmin` command. The `--client-cert` parameter points to the orderer certificate issued by the TLS CA, and `--client-key` refers to the orderer private key issued by the TLS CA.  
+
+    For example:
+    ```
+    osnadmin channel list –-orderer HOST1:7081 --ca-file organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/cacerts/tls-ca-cert.pem --client-cert organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/cert.pem --client-key organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/keystore/tls-key.pem.pem
+    ```
+
+    The output of this command looks similar to:
+
+    ```
+    {
+      "systemChannel": {
+          "name": "systemchannel",
+          "url": "/participation/v1/channels/systemchannel"
+      },
+      "channels":[
+          {
+            "name": "channel1",
+            "url": "/participation/v1/channels/channel1"
+          }
+      ]
+    }
+    ```
+    Run `osnadmin system-channel remove` to remove the system channel from the node configuration:
+
+    ```
+    osnadmin system-channel remove --orderer [ORDERER_ADMIN_LISTENADDRESS] --channelID systemchannel --ca-file [TLS_CA_ROOT_CERT] --client-cert [OSN_TLS_SIGN_CERT] --client-key [OSN_TLS_PRIVATE_KEY]
+    ```
+
+    For example:
+    ```
+    osnadmin system-channel remove --orderer HOST1:7081 --channelID systemchannel --ca-file organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/cacerts/tls-ca-cert.pem --client-cert organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/cert.pem --client-key organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/keystore/tls-key.pem.pem
+    ```
+
+    When you rerun the `osnadmin channel list` command, you can verify that the system channel was removed:
+    ```
+    osnadmin channel list –-orderer [ORDERER_ADMIN_LISTENADDRESS] --ca-file [TLS_CA_ROOT_CERT] --client-cert [OSN_TLS_SIGN_CERT] --client-key [OSN_TLS_PRIVATE_KEY]
+    ```
+
+    Examine the output of the command to verify that the system channel is no longer listed:
+    ```
+    {
+      "channels":[
+          {
+            "name": "channel1",
+            "url": "/participation/v1/channels/channel1"
+          }
+      ]
+    }
```

Comment:
This output isn't accurate. There should be a "systemchannel" entry that shows

"systemchannel": null 

after removing it.

Reply:
Updated to:
```
{
      "systemChannel": null,
      "channels":[
          {
            "name": "channel1",
            "url": "/participation/v1/channels/channel1"
          }
      ]
    }
```